pkg/lib/cockpit-components-inline-notification.tsx

```diff
@@ -66,11 +79,12 @@ export class InlineNotification extends React.Component {
         }
         const extraProps = {};
         if (onDismiss)
+            // @ts-expect-error: PF bug: actionClose not defined in Alert
             extraProps.actionClose = <AlertActionCloseButton onClose={onDismiss} />;
 
         return (
             <Alert variant={type || 'danger'}
-                isLiveRegion={this.props.isLiveRegion}
+                isLiveRegion={this.props.isLiveRegion ?? false}
```

Comment:
This added line is not executed by any test.

Reply:
They are covered in #20726.